test/test_pubsub.py

```diff
@@ -2,20 +2,28 @@
 #
 # Copyright (C) 2022 Collabora Limited
 # Author: Michal Galka <michal.galka@collabora.com>
+# Author: Alexandra Pereira <alexandra.pereira@collabora.com>
 
+from unittest import result
+from attr import attributes
 from api.pubsub import PubSub
 import fakeredis.aioredis
+import json
 import pytest
 
 
-@pytest.fixture()
-def mock_pubsub(mocker):
+def prepare_pubsub_mock(mocker):
     pubsub = PubSub()
     redis_mock = fakeredis.aioredis.FakeRedis()
     mocker.patch.object(pubsub, '_redis', redis_mock)
     return pubsub
 
 
+@pytest.fixture()
+def mock_pubsub(mocker):
+    return prepare_pubsub_mock(mocker)
+
+
```

Comment:
Is there a reason why we need a `prepare_pubsub_mock()` function instead of keeping the code in the fixture?

Reply:
it makes sense, no special reason and fixed